### PR TITLE
docs: Broken links and Documentation review

### PR DESCRIPTION
**Pull request by nickrandolph**
## PR Type

What kind of change does this PR introduce?
- Documentation content changes

### CODE REVIEW

File: doc/Learn/Tutorials/Authentication/HowTo-Cookies.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+uid: Learn.Tutorials.Authentication.HowToCookieAuthorization
+---
+# How-To: Using Cookies to Authorize
+
+`Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```
Comment (amelield) [inline doc/Learn/Tutorials/Authentication/HowTo-Cookies.md:6]: It is recommended to do what with one of your build? I do not understand that sentence.

File: doc/Learn/Tutorials/Authentication/HowTo-MsalAuthentication.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+uid: Learn.Tutorials.Authentication.HowToMsalAuthentication
+---
+# How-To: Get Started with MSAL Authentication
+
+`Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```
Comment (amelield) [inline doc/Learn/Tutorials/Authentication/HowTo-MsalAuthentication.md:6]: Same question. Maybe a word is missing?

File: doc/Learn/Tutorials/Authentication/HowTo-OidcAuthentication.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+uid: Learn.Tutorials.Authentication.HowToOidcAuthentication
+---
+# How-To: Get Started with Oidc Authentication
+
+`Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```
Comment (amelield) [inline doc/Learn/Tutorials/Authentication/HowTo-OidcAuthentication.md:6]: It is recommended to use one?

File: doc/Learn/Tutorials/Authentication/HowTo-WebAuthentication.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+uid: Learn.Tutorials.Authentication.HowToWebAuthentication
+---
+# How-To: Get Started with Web Authentication
+
+`Uno.Extensions.Authentication` provides you with a consistent way to add authentication to your application. It is recommended to one of the built in `IAuthenticationService` implementations. This tutorial will use the custom authorization to validate user credentials.
```
Comment (amelield) [inline doc/Learn/Tutorials/Authentication/HowTo-WebAuthentication.md:6]: Same

**Comment by lukeblevins** [comment]
Just fyi, I just attempted to resolve conflicts in:
* /testing/TestHarness/TestHarness.Shared/App.xaml.cs
* /src/Uno.Extensions.Localization.UI/LocalizationService.cs

@nickrandolph This PR appears to fix broken links and the issues mentioned by @amelield above should not be public, so we should be able to go ahead with it.

**Comment by lukeblevins** [comment]
Fixed the typos spotted by @amelield

**Review by lukeblevins — APPROVED** [review]
lgtm, let me know if there are any more blockers for this one

**Pull request merged**